propagator/jaeger/lib/opentelemetry/propagator/jaeger/text_map_injector.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+module OpenTelemetry
+  # Namespace for OpenTelemetry propagator extension libraries
+  module Propagator
+    # Namespace for OpenTelemetry Jaeger propagation
+    module Jaeger
+      # Injects context into carriers
+      class TextMapInjector
+        DEFAULT_FLAG_BIT = 0b0
```

Comment:
Any reason not to represent these as base16 like the spec: https://www.jaegertracing.io/docs/1.21/client-libraries/#propagation-format